docs/walk-through/about-hera.md

```diff
@@ -1,4 +1,4 @@
-# About
+# About Hera
```

Comment:
Maybe this section can emphasize, in bold, that "Anything you can express in Argo Workflows you can express in Hera!"? :)

Reply:
I also wanted to bold the `hello-world.md`'s "Using Any Function on Argo" header 👀